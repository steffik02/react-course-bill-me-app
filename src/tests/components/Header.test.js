// import ReactShallowRenderer from "react-test-renderer/shallow";
//react test renderer allows us to render our compments inside of JS code and assert someting about what got rendered
import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('h1').text()).toBe('Bill.Me');

    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});