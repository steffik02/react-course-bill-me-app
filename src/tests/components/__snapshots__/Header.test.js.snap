// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Bill.Me
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Create Expense
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
